pontoon/api/serializers.py

```diff
@@ -254,14 +246,13 @@ class Meta:
         ]
 
     def get_translation_text(self, obj):
-        request = self.context.get("request")
-        locale = request.query_params.get("locale") if request else None
-
-        if not locale:
-            return None
+        translation = (
+            obj.filtered_translations[0].text
+            if hasattr(obj, "filtered_translations") and obj.filtered_translations
+            else None
+        )
```

Comment:
(nitpick, style) Split in two as GitHub can't handle them in one:
```suggestion
        if hasattr(obj, "filtered_translations") and (ft := obj.filtered_translations):
            return ft[0].text
```